.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  push:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5
```

Comment:
Can we use 3.8 for this?  I'm not aware of any issues that would cause a package created by a newer Python version to not work correctly in an older one.

Reply:
Done.